app/src/components/cards/AccountDetailsCard/AccountDetailsCard.tsx

```diff
@@ -171,9 +171,11 @@ const AccountDetailsWrapper = styled.div`
 
 const HeadContentContainer = styled(HeadContentWrapper)<{
   isTruncated?: boolean;
+  isMobile?: boolean;
 }>`
   margin: 0;
-  margin-bottom: ${({ isTruncated }) => (isTruncated ? '0.5rem' : '5rem')};
+  margin-bottom: ${({ isTruncated, isMobile }) =>
+    isTruncated || isMobile ? '0.5rem' : '5rem'};
```

Comment:
Please refactor these styles to use media queries instead of `isMobile`

Reply:
Refactored!